src/intents.cs

```diff
@@ -602,8 +604,10 @@ public enum INSendPaymentIntentResponseCode : nint {
 		FailureInsufficientFunds,
 		FailureNoBankAccount,
 		[Introduced (PlatformName.iOS, 11, 0)]
-		[Introduced (PlatformName.MacOSX, 10, 13, PlatformArchitecture.Arch64)]
```

Comment:
we're sure the above is not in macOS ?

Reply:
I guess it make sense as the enum is decorated with `[Unavailable (PlatformName.MacOSX)]`

Reply:
Yup that was my bad :)